cypress/e2e/smoke/conditions-tab-ui.cy.js

```diff
@@ -31,5 +33,11 @@ context("Conditions panel", () => {
             map.getKeyContainer().should('contain','Risk Level');
             map.getMapKeyToggle().should('contain','Show Tephra');
         });
+        it('verity LavaCoder shows correct elements',()=>{
+            modelOptions.getModelOptionsMenu().click();
+            modelOptions.selectUnitOption('LavaCoder');
+            conditionsTab.getLavaFrontHeightWidget().should('be.visible');
+            conditionsTab.getVolumeOfLavaWidget().should('be.visible');
+        });
```

Comment:
This is fine, but I think there's also a way to specify the simulation via url parameter which would obviate the need to navigate the model options menu.

Reply:
I'm going to put a TODO in about this but I'm not going to change it for now.